git/functions/gdel

```diff
@@ -16,5 +16,6 @@ EOF
 }
 
 # `$@` holds all arguments
+git branch --unset-upstream "$@" > /dev/null 2>&1
```

Comment:
`gdel` should be a variadic function so that someone can run `gdel branch1 branch2 ...` to delete multiple branches at once. Removing the redirection to `/dev/null` and testing with multiple branches seems to fail:
![image](https://user-images.githubusercontent.com/9750687/70589814-943f5400-1b85-11ea-9b30-46ff585320e2.png)

I think we'll have to loop through the branches and run this command on each one in turn. Something like this:

```suggestion
for branch in "$@"; do
  git branch --unset-upstream "$branch" > /dev/null 2>&1
done
```